### PR TITLE
Explaining what costs what

### LINKED ISSUE #36: Be clear about admin fees and related rules
Instructor training needs to be much clearer about use of the 'instructors' mailing list, admin fees, etc.

----

Our policy with self-organized workshops is that you are not allowed to use central resources to find instructors unless you are paying our workshop administration fee for the workshop. If you would like to pay the workshop administration fee we will be happy to help you find instructors. This instructor matchmaking is exactly why we employ folks (at a cost to us) that develop software for workshop organizing and are able to match the most appropriate instructor to the workshop and its audience.

### PR DESCRIPTION
**Pull request by gvwilson**
Uses direct quotes from SWC and DC sites.

Closes #36.

### CODE REVIEW

File: _episodes/14-carpentries.md [hunk 1]
```diff
@@ -96,6 +96,62 @@ and eventually added to our operations guide.
...
+>
+> If you are a not-for-profit, such as a university or government lab,
+> the Software Carpentry Foundation will organize a workshop for you
+> (not including instructor travel and accommodation costs) for $2500 USD.
+>
+> **For-Profit Institution: $10000**
+>
+> If you are a for-profit institution, such as a company,
+> the Software Carpentry Foundation will organize a workshop for you
+> (not including instructor travel and accommodation costs) for $10,
```
Comment (abought) [inline _episodes/14-carpentries.md]: Is this number missing a 0?

(Should be `$10,000`)

Comment (gvwilson) [inline _episodes/14-carpentries.md]: Thanks - fixed.

File: _episodes/14-carpentries.md [hunk 2]
```diff
@@ -96,6 +96,62 @@ and eventually added to our operations guide.
 > What situations might come up that these *don't* answer?
 {: .challenge}
 
+### What Costs What?
+
+Quoting the [Software Carpentry workshop request page][swc-request]:
+
+> Our instructors are volunteers,
+> and so are not paid for their teaching,
+> but host sites are required to cover travel and accommodation costs for any instructors visiting from out of town.
```
Comment (ChristinaLK) [inline _episodes/14-carpentries.md]: Bold/emph this line

**Comment by ChristinaLK** [comment]
Seeing as this quotes directly from the websites, I'm not sure it needs further review.  I do think it could be clearer that organizing your own means finding your own instructors, and what *is* included in a workshop being organized for you, but that would be a website change, not here.

**Pull request merged**